pkg/dao/crd/dao.go

```diff
@@ -100,6 +100,25 @@ func (d *Dao) SetSpec(id string, spec *bundle.Spec) error {
 	return err
 }
 
+// updateSpec - update a spec for an id in the kvp API and overwrite the
+// existing spec.
+func (d *Dao) updateSpec(id string, spec *bundle.Spec) error {
```

Comment:
Regarding the other comment about not having the update, I think that this should be apart of SetSpec.

We already override SetXXXX to include updates with both instances. I think that we should probably just do the same thing here.